pkg/lib/page.scss

```diff
@@ -191,6 +191,14 @@ html:not(.index-page) body {
   margin-block: var(--offset);
 }
 
+// RTL: Pass through computer-generated text as-is
+// Important: Only apply this to elements that are or can be inline.
+// In many cases, it may be better to wrap output text with a <bdi> element
+.ct-rtl-passthrough {
+  display: inline;
+  unicode-bidi: plaintext;
```

Comment:
"rtl-passthrough" is a bit confusingly named. I suppose it means "passthrough in an RTL context", not "pass through RTL text".

The "computer-generated" text will often be LTR, but we often also pass through translated messages from the system, which could be RTL. I just read https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi and it seems it uses the unicode markers contained in the string itself to decide that. I'm not actually sure that the strings will always have that, at least not if they are explicit Unicode control characters. If it determines that based on the character range, that's fine of course.

A good test case may be the motd box -- putting in LTR or RTL text (or both), and showing that on an LTR or RTL Cockpit language. We could add an integration pixel test for all four cases.

Another good test case could be the hwinfo page. Things like "model name" will almost certainly be Latin/English/LTR, and we can test that on an RTL page.